package.json

```diff
@@ -56,7 +56,7 @@
   "binary": {
     "module_name": "bcrypt_lib",
     "module_path": "./lib/binding/",
-    "host": "https://github.com",
-    "remote_path": "/kelektiv/node.bcrypt.js/releases/download/v{version}/"
+    "host": "https://github.com/kelektiv/node.bcrypt.js/releases/download/",
+    "remote_path": "v{version}"
```

Comment:
Understood.

I am going to manually track down and upload the windows and linux assets for the release to github to unblock whomever I can while we continue to iterate.

It might be nice to just make a PR adding osx support to travis and node v7 so it will build the assets for us now that I can upload.